hackage-security/hackage-security.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       1.12
 name:                hackage-security
-version:             0.6.2.1
+version:             0.6.2.2
```

Comment:
If you bump the version, you should delete the `x-revision` field, (or set the revision to `0` I suppose, but I never tried this).

Reply:
Done. Thank you.

Reply:
its not clear to me why there's an x-revision field in the cabal file in the repo at all? typically x-revision is supposed to be managed for you by hackage server.

Reply:
Makes it easier to automatically upload the revision with https://github.com/hackage-trustees/hackage-cli